cmd/sifnoded/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN go mod download
 
 COPY . .
 
+ENV FEATURE_TOGGLE_MARGIN_CLI_ALPHA=1
```

Comment:
@canercandan this needs to be removed.

Reply:
@timlind good catch Tim, updating it now thanks